packages/cmf/src/components/ErrorFeedBack/ErrorFeedBack.component.test.js

```diff
@@ -24,18 +28,7 @@ describe('Component ErrorFeedBack', () => {
 				stack: 'Error stack',
 			},
 		];
-		const wrapper = mount(<Component errors={errors} fullPage />);
-		expect(wrapper.find('.col-md-6').props().style.marginTop).toBe(200);
-	});
-	it('should render react fragment in fullPage mode', () => {
-		const errors = [
-			{
-				message: 'Error message',
-				name: 'Error name',
-				stack: 'Error stack',
-			},
-		];
-		const wrapper = shallow(<Component errors={errors} />);
-		expect(wrapper.find('Fragment').length).toBe(1);
+		const wrapper = render(<Component errors={errors} fullPage />);
+		expect(wrapper.container.querySelector('.col-md-6').style.marginTop).toBe('200px');
```

Comment:
I didn't get the added value of that test

Reply:
lets remove it !